src/Nethermind/Nethermind.Synchronization/FastBlocks/BarrierSyncFeed.cs

```diff
@@ -32,7 +32,7 @@ public abstract class BarrierSyncFeed<T> : ActivatedSyncFeed<T>
     protected bool WithinOldBarrierDefault => _specProvider.ChainId == BlockchainIds.Mainnet
         && _barrierWhenStarted == DepositContractBarrier
         && LowestInsertedNumber <= DepositContractBarrier
-        && LowestInsertedNumber > DepositContractBarrier - GethSyncLimits.MaxBodyFetch; // this is intentional. using this as an approxamation assuming a minimum of 1 receipt in per block in case of receipts
+        && LowestInsertedNumber > DepositContractBarrier - 5_000; // this is intentional. this is a magic number as to the amount of possible blocks that had been synced. We noticed on previous versions that the client synced a bit below the default barrier by more than just the GethRequest limit (128).
```

Comment:
lets make it round (4096 or 8192)? Maybe introduce named constant for it?

Reply:
sure